go.mod

```diff
@@ -0,0 +1,81 @@
+module github.com/openshift/builder
+
+go 1.12
+
+require (
+	github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
+	github.com/certifi/gocertifi v0.0.0-20180905225744-ee1a9a0726d2 // indirect
+	github.com/containerd/continuity v0.0.0-20190426062206-aaeac12a7ffc // indirect
+	github.com/containers/buildah v1.11.3
+	github.com/containers/image v3.0.2+incompatible
```

Comment:
I would expect this line to go away or be an `// indirect` dependency.

After making the two other edits, `go mod vendor` removes the non-v4 copy.

Reply:
Looking at e7c2cd6b6ff6f7d261c27c92a967e4cb19ec85ea , please run `go mod vendor` as well.

Reply:
I ran `go mod vendor` multiple times through the process.

Do you mean run `go mod vendor` on top of my last commit here?

Reply:
Yes, on top of the one that removes the last `/v4/` bits. (Or, being pedantic, that part should be moved from the “seed containers/image with…” commit into the  move containers/image imports to v4” one.

After checking out the commit I quoted above, `go mod vendor` removes `vendor/github.com/{containers/image,docker/libtrust}` for me.

Reply:
thanks for clarifying ... did for me as well

and things work locally ... will push momentarily